### PR TITLE
fix: camera process fail to start

### PR DESCRIPTION
**Pull request by bobby-palko**
- Fixes the init of the camera child process so it calls super and initializes without crashing
- Adds a print statement to the exception instead of just passing
- Formatting cleanup, such as removing unused code and shortening lines that are over the pylint recommended length

### CODE REVIEW

File: rear_rider_device/camera_child_proc.py [hunk 1]
```diff
@@ -1,48 +1,61 @@
...
\ No newline at end of file
+        pass
+
+    def no_on_handler(self, on_command, err):
+        """
+        An empty function.
+
+        An inheriting class can override this method to customize what happens when the
+        child process is done communicating with the parent process.
+        """
```
Comment (jon-dez) [inline rear_rider_device/camera_child_proc.py:60]: This doc string is invalid for this method. I would suggest to remove it. The method is originally defined in `ChildProcess` but I didn't add a doc string. Let me know if you would like me to create a PR for updating the `ChildProcess` doc strings.

**Review by jon-dez — CHANGES_REQUESTED** [review]

File: rear_rider_device/camera_child_proc.py (hunk 1 above)
Comment (jon-dez) [reply rear_rider_device/camera_child_proc.py:60]: The preview doesn't show it, but this is for the `no_on_handler` method, which is called when a does not have a handler for a specified message (on_command) type.

Comment (bobby-palko) [reply rear_rider_device/camera_child_proc.py:60]: yeah, we should update the `ChildProcess` doc strings too. I'll update this one to reflect what it's for 👍🏻

**Review by jon-dez — APPROVED** [review]
Looks good!

**Pull request merged**